QSB/default-config.json

```diff
@@ -3,6 +3,6 @@
   "settings": {
     "defaultServerIP": "localhost",
     "port": 7777,
-    "debugMode": false
+    "debugMode": true
```

Comment:
sure?

Reply:
just for while i'm developing it. for the final commit i change it back